src/shared_gui_components/MeasureManager.cpp

```diff
@@ -607,8 +607,10 @@ boost::optional<measure::OSArgument> MeasureManager::getArgument(const measure::
 
   } else if (type.value() == measure::OSArgumentType::Path) {
 
+    // TODO: aside from the fact that these arguments are weird / incorrectly named, neither the BCL-gem schema nor OS SDK actually handle them so
+    // they are not even inside the measure.xml
     bool isRead = argument.get("is_read", Json::Value(false)).asBool();
```

Comment:
Does `isRead` mean that the file should be read from the path vs written to the path?

Reply:
I think that was the intent. not sure